src/tests/Microsoft.PowerFx.Core.Tests/Microsoft.PowerFx.Core.Tests.csproj

```diff
@@ -26,6 +26,7 @@
     <Title>Microsoft Power Fx Compiler Tests Core</Title>
     <Description>The core test suite for all Microsoft Power Fx Compiler targets</Description>
     <Summary>The core test suite for all Microsoft Power Fx Compiler targets</Summary>
+    <TargetFrameworks>net5</TargetFrameworks>
```

Comment:
This overrieds core3.1 specified on lines 7 & 8. Is there a reason for it?

Reply:
ah, i see https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.moduleinitializerattribute?view=net-6.0

Reply:
We can't actually make this test project .NET 5, there are some other test projects external to this repo that depend on it that are still on Core 3.1. Take a look at either of the top two answers here for doing assembly-wide initialization in xunit

https://stackoverflow.com/questions/13829737/run-code-once-before-and-after-all-tests-in-xunit-net/53143426#53143426